includes/Hooks.php

```diff
@@ -376,6 +385,45 @@ public function onLocalUserCreated( $user, $autocreated ) {
 		if ( !$user->isTemp() && !$autocreated ) {
 			$this->userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
 		}
+		if ( !$autocreated ) {
```

Comment:
todo: check what happend here because changes in this file are not present in the cherry-picked commit https://github.com/Wikia/mediawiki-extensions-Thanks/pull/4/commits/dcbbee36601a823c8eb9e36f8f7c12c403ed1f69